recipes/sdl/all/conanfile.py

```diff
@@ -379,6 +391,9 @@ def package_info(self):
 
         postfix = "d" if self.settings.os != "Android" and self.settings.build_type == "Debug" else ""
 
+        if self.version >= "2.0.24" and is_msvc(self) and not self.options.shared:
+            postfix = "-static" + postfix
```

Comment:
The static suffix is only applied on the main SDL2 target.
SDL2_test and SDL2main are always static, so don't need this suffix.

Reply:
omg auhahuahuahuahuahuauh, thanks.